steward/src/commands/schedule_corks/aave_cellar/shutdown.rs

```diff
@@ -28,25 +28,20 @@ pub struct ShutdownCmd {
     #[clap(short = 'b', long)]
     height: u64,
 
-    ///Set to true if you want to shutdown Aave Cellar.
-    #[clap(short = 's', long)]
-    shut_down: bool,
-
     /// Set to true if you want to exit current position.
     #[clap(short = 'e', long)]
-    exit_position: bool,
+    empty_position: bool,
 }
 
 impl Runnable for ShutdownCmd {
     fn run(&self) {
         let config = APP.config();
 
         abscissa_tokio::run_with_actix(&APP, async {
-            let call = SetShutdownCall {
-                shutdown: self.shut_down,
-                exit_position: self.exit_position,
+            let call = InitiateShutdownCall {
+                empty_position: self.empty_position,
             };
-            let encoded_call = AaveV2StablecoinCellarCalls::SetShutdown(call).encode();
+            let encoded_call = AaveV2StablecoinCellarCalls::InitiateShutdown(call).encode();
```

Comment:
Based on the new ABI, there also needs to be a `LiftShutdown` scheduled cork.

Reply:
Indeed. I left this for future work. Do you think I should go ahead and add it?

Reply:
Since there is an existing shutdown scheduled cork, and this contract update replaces it with only half the functionality it had before, it's probably an appropriate time to update it.